pkgs/development/tools/azure-static-sites-client/default.nix

```
@@ -0,0 +1,116 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, curl
+, icu70
+, libkrb5
+, lttng-ust
+, openssl_1_1
+, zlib
+, azure-static-sites-client
+  # "latest", "stable" or "backup"
+, versionFlavor ? "stable"
+}:
+let
+  icu = icu70;
+  versions = lib.importJSON ./versions.json;
+  flavor = with lib; head (filter (x: x.version == versionFlavor) versions);
+  fetchBinary = runtimeId: fetchurl {
+    url = flavor.files.${runtimeId}.url;
+    sha256 = flavor.files.${runtimeId}.sha;
+  };
+  sources = {
+    "x86_64-linux" = fetchBinary "linux-x64";
+    "x86_64-darwin" = fetchBinary "macOS";
+  };
+  isOldVersion = lib.versionOlder flavor.buildId "1.0.021611";
+in
+stdenv.mkDerivation rec {
+  name = "${pname}-${versionFlavor}-${version}";
+  pname = "StaticSitesClient";
+  version = flavor.buildId;
```

```suggestion
  pname = "${pname}-${versionFlavor}";
  version = flavor.buildId;
```

name should always be ``${pname}-${version}``. Now we can no longer substitute pname but thats okay.

---

Done.